api/configs/configs.py

```diff
@@ -0,0 +1,32 @@
+import importlib
+
+'''
+**READ ME**
+Never put any sensitive information here i.e. credentials, secrets, etc.
+'''
+
+class Config(object):
```

Comment:
With Python 3, we don't have to inherit from `object`. So this can just be `class Config:`.